css/_welcome_page.scss

```diff
@@ -4,90 +4,70 @@ body.welcome-page {
 }
 
 .welcome {
+    background-color: $welcomePageHeaderBackground;
+    display: flex;
+    flex-direction: column;
     font-family: $welcomePageFontFamily;
-    height: 100%;
+    justify-content: space-between;
+    min-height: 100vh;
     position: relative;
 
     .header {
         align-items: center;
-        background: $welcomePageHeaderBackground;
         display: flex;
         flex-direction: column;
+        min-height: fit-content;
         overflow: hidden;
         position: relative;
         text-align: center;
 
         .header-text {
             display: flex;
             flex-direction: column;
-            justify-content: space-around;
-            margin-top: 120px;
-            margin-bottom: 20px;
+            margin-top: 156px;
+            margin-bottom: 36px;
             max-width: calc(100% - 40px);
-            min-height: 286px;
-            width: 645px;
+            width: 800px;
+            z-index: $zindex2;
         }
 
         .header-text-title {
             color: $welcomePageTitleColor;
-            font-size: 48px;
+            font-size: 40px;
+            font-weight: 700;
             letter-spacing: -1px;
-            line-height: 58px;
-            margin-bottom: 20px;
+            line-height: 48px;
+            margin-bottom: 16px;
```

Comment:
After playing with font sizes and styles I believe the optimal ones to be: 
font-size: 2.5rem (if we're to use rems, which would be cool)
font-weight: 500;
letter-spacing: 0;
line-height: 1.18;